package.json

```diff
@@ -15,6 +15,7 @@
   "license": "MIT",
   "dependencies": {
     "@azure/storage-blob": "^12.1.1",
+    "@covid-modeling/api": "0.8.0",
```

Comment:
0.8.1?

Reply:
0.8.0 and 0.8.1 are the same (just published differently).

Reply:
Though, I'll change it just to be sure.